text/0000-discord.md

```diff
@@ -0,0 +1,170 @@
+
+Out of frustration with Slack's disappearing messages, the Ember.js core team set up a Discord server to evaluate if it might be a better fit for open source communities.
+
+While this was a public Discord server that anyone could sign up for, its existence was not widely publicized because we were unsure if Discord was the right solution.
+
+Over time, the core team and many contributors gravitated towards the Discord, finding that it served our needs better. Because of how valuable the Slack instance is, no one wanted to propose a move to Discord until a plan (like, say, this RFC) could be put in place.
+
+Unfortunately, this state of affairs has had several undesirable outcomes.
+
+First, it has caused many of the most prolific contributors to be less active in Slack. This may give the appearance of stagnation or disinterest, when momentum on Ember has never been higher. It robs lurkers of the ability to become contributors if a good opportunity to help pops up. And it prevents some of the most experienced members of the community from being around to help answer questions they might have an off-hand answer to.
+
+Second, and perhaps worst of all, it undermines the transparency and open governance that we have worked hard to create. Our bar is higher than just making it possible to contribute—we go out of our way to actively welcome and encourage everyone to participate, learn and contribute.
+
+## Detailed design
+
+### Transition Plan
+
+We will need these things to transition the community smoothly:
+
+- a period of time when we use both chat platforms during the transition, put the equivalent Discord channel information in the Slack channel topic
+- a clear guide (with illustrations)
+
+#### Initial Setup
+
+Because Discord has fine-grained controls, we will be able to implement categories for chats.
+
+We intend to have the "welcome" channel as the initial channel for everyone who joins the Discord server. This channel will be read-only and will list the rules for the Discord server.
+
+**Verification Level**
+Initially, we will be implementing the "low" verification level, which means users will need to have a verified email on their Discord account. If this proves to be too easy of a target for spammers, we will implement a higher level of verification (levels include amount of time a user has to be a verified member of the server before they can post).
+
+**Explicit Content Filter**
+Since this is a public Discord server, we will be setting an explicit content filter- it will scan messages from all members without a role.
+
+**Categories and Channels**
+Community members will then have the option of visiting the "setup" channel and learning more about fine-grained controls, such as:
+
+- notifications
+- muting a channel
+- muting a category
+
+Because our goal is transparency, all channels will be visible. A lock icon will display if the user does not have the role necessary to join that channel.
```

Comment:
Can visibility be defined here for those folks unfamiliar with discord?  Will the content of the channels be visible regardless of the user role?